cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -151,8 +151,9 @@ def is_displayed(self):
 
 
 class DetailsEntitiesCatalogItemView(View):
-    custom_image = FileInput("upload_image")
-    smart_management = SummaryTable("Smart Management")
```

Comment:
Q: why we are removing `smart_management`?

Reply:
thanks for pointing. I forgot to re-add it